crates/turbopack-tests/tests/execution/turbopack/tree-shaking/.namespace-object-identity/input/index.js

```diff
@@ -0,0 +1 @@
+import "../../../side-effects-optimization/namespace-object-identity/input/index.js";
```

Comment:
@sokra This test is wrong because this test wants module objects to be identical, and have no cyclic refernce which cannot be true under the current tree shaking design